uint8 control_ability 
---
bool result
